packages/app/src/styles/types.ts

```diff
@@ -121,3 +121,9 @@ export interface AffineThemeCSSVariables {
 declare module '@emotion/react' {
   export interface Theme extends AffineTheme {}
 }
+
+declare global {
+  interface Window {
+    CLIENT_APP?: boolean;
+  }
+}
```

Comment:
It's not appropriate to put the global declaration in `styles/types.ts`. You should create a new `types.ts` at src dir.

Reply:
Sorry I didn't notice the `/styles`, just searched for a types.ts to add it...